controlpanel/api/migrations/0008_remove_unused_role_team_models.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.3 on 2019-07-18 11:14
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
shouldn't there be a migration to remove `Group` too?

Reply:
I'm not sure about messing with models from `contrib.auth` - and I think we may want to use them in the future.

Reply:
oh right, I thought it was one of our models.